fastlane/lanes/screenshots.rb

```diff
@@ -143,7 +103,7 @@
 
     locales = ALL_LOCALES
       .select { |hsh| hsh[:promo_config] != false }
-      .map {| hsh | [ hsh[:glotpress], hsh[:google_play] ]}
+      .map { |h| h[:google_play] }
```

Comment:
⚠️  This might be a copy/paste error, as it seems it was returning an array of `[glotpress, googleplay]` items before but is now using a single one.

I think that's when I tried to fix the incorrect list of locales and then copy/pasted the fix everywhere for consistency and missed the subtle difference here.

(Can't wait to have time to finish my work on the `LocaleHelper` in the `release-toolkit` at some point in the future to have all this locale manipulation simplified!)

Reply:
Fixed in 1dbc2c995a7c85f64bef883054f3fa77a046c08c